### ISSUE TITLE
Final Task Early Submission Feedback - Missing sources in STU2_SAR_TaskFinal.json for final submission

### ISSUE THREAD

**Issue by Cazael** [rcghpge/stu-uta-lunar-sar-2024#2]
Hi Lunar Mavericks, 

I started going through your submission to provide some early feedback before final submission judging and noticed that your Final Task json file doesn't seem to be properly setup. 
"[STU2_SAR_TaskFinal.json](https://github.com/rcghpge/STU_Lunar_SAR_LunarMavericks/blob/8464282291c19d311a317fd7c66f57392655e896/sims/STU2_SAR_TaskFinal.json)"

For example, it is currently configured that you are using the python file ExampleMM_TaskFinal.py for your mission manager. However, this example mission manager hasn't been edited in any way.
"[ExampleMM_TaskFinal.py](https://github.com/rcghpge/STU_Lunar_SAR_LunarMavericks/blob/8464282291c19d311a317fd7c66f57392655e896/ExampleMM_TaskFinal.py)"

Please let me know if there are any files that may be missing or perhaps you just need to assign the files accordingly in the TaskFinal.json because the sources being used right now are the default ones. 

We will keep looking through the rest of the code to see if we can provide any more useful feedback but I wanted to notify you about that issue as soon as possible. Hope to hear back soon, good luck!

**Comment by rcghpge** [comment]
Hey Cazael, pretty much most of the team was unable to install the software. So it kind of boiled down to about 3 of us. If that helps explain the gist of our repo. Are we still allowed to update code in our repository?

**Comment by connorjak** [comment]
How y'all have uploaded duplicates of all the files instead of overwriting the existing files makes things quite difficult to find. Not impossible though. It seems to us that your final submission is unchanged from the starter code. Is this correct? If it's not, please help us find the added functionality from your team.

It's well past the deadline so y'all won't be able to make additional functionality additions. I see your feedback in the submission form; many of the other teams had time issues with completing the challenge as well.

What kinds of issues did your team run into in installing STPRO? Incompatible OS / not meeting system requirements? Errors in install process? Unable to run any sims? Unable to run the STU Lunar SAR sim in particular?

**Comment by rcghpge** [comment]
Hey Connor, I'll see if we can pull up back-up copies of the STU_SAR starter codebase in the morning. Some of our implementations did not make it into the final submission. There were a lot of software updates leading up to the final submission deadline and the time constraints just wouldn't allow us enough of a window to implement everything between changes in versioning of the STPro stack and the STU_SAR codebase. I guess a combination of the 2.

**Comment by rcghpge** [comment]
Update: I copied what we have onto a folder and what was permissibly allowed without permission issues. GitHub does not allow folder uploads (eg file formats etc.) and it looks like cloud providers have permission clauses/issues with some software stacks so there were a lot of library and software packages/dependencies in a couple of folders that I left out. We had begun trying to implement ML/DL systems into the build-out but did not have the time by the final/special submission deadline balancing the challenge among version updates, school, and other obligations.  I was able to host a copy of most of what should have our team's codebase for the 2024 STU SAR Challenge on Google Drive. Here is the link: https://drive.google.com/drive/folders/1RiKeO72wLWke27Pbi7odPpj6zYqoQA7i?usp=sharing

I was trying to verify if 1 of the Python Scripts had a Tensorflow-Keras library imported in 1 of the MM scripts or the sort. Could not validate the information. We are about at our finals week juncture here at UT Arlington so we are wrapping up the year for the holidays
